_posts/en/newsletters/2019-02-05-newsletter.md

```diff
@@ -0,0 +1,283 @@
+the following topics especially interesting from a Bitcoin perspective,
+although we encourage anyone who wants to learn more to look at the
+[transcripts][] provided by Bryan Bishop or the videos provided by
+the organizers ([day 1][], [day 2][], [day 3][]).
+
+- **Accumulators for blockchains** presented by Benedickt Bünz
+  ([transcript][accumulators txt], [video][accumulators vid]).  Bitcoin
+  full nodes maintain a ledger (called the UTXO set) that stores the
+  ownership information for every currently-spendable grouping of
+  bitcoins.  Currently, that ledger contains over 50 million entries
+  and uses about 3.8 gigabytes of disk space.  This allows nodes who
+  receive a transaction to look up the bitcoins being spent by that
+  transaction in the UTXO set, retrieve their ownership information, and
+  verify that against the signature and other witness data provided in
+  the spend.
+
+    But what if we also asked the spender to provide a copy of the
+    ownership information in their transaction along with a
+    cryptographic proof that the the information is actually in the UTXO
+    set?  In that case, we wouldn't need to store the whole set, we'd
+    only need to store a commitment to a set we knew was accurate and
+    that could be referenced by accurate proofs.  RSA accumulators are
+    one idea (among several others) for how to create that commitment
+    and its related proofs.  Using an accumulator, the size of the UTXO
+    commitment that nodes would store on disk would be tiny compared to
+    the full state.  Transactions would increase in size due to needing
+    to provide ownership data and a proof that they were part of the
+    current UTXO set, but the size increase would be modest (assuming
+    current typical transactions, about 70 bytes of ownership
+    information per input plus a proof that could be aggregated down to
+    about 325 bytes per block).
+
+    Other considerations affect the suitability of accumulators to
+    the task, including being relatively new to cryptography,
+    requiring either a well-studied trusted setup or a
+    more novel untrusted setup, and potentially making blocks take
+    longer to verify than the current system given that accumulator
+    verification is about 100x slower than alternative systems using
+    merkle trees.  In a new development compared to a previous
+    version of this talk given at Scaling Bitcoin 2018, the speaker
+    describes a potential major speedup to practical verification.
+
+    In summary, RSA accumulators remain an interesting area of research
+    in to how to reduce full node storage requirements.  At present,
+    it's debatable whether this would provide much advantage for
+    Bitcoin---with today's disk sizes, few users are probably
+    discouraged by having to store a mere 4 GB---but it could
+    significantly improve usability if block sizes were to grow
+    considerably or if Bitcoin were to change to a more trusted model
```

Comment:
Nit: I'm a little wary of talking about a 'change to a more trusted model'. I would expect any 'assume UTXO set' security model would be optional, and full node operators would have the option of fully validating from genesis if they wanted.